### PR TITLE
Change uses of `git add` to `git stage`

### LINKED ISSUE #62: Replace uses of `git add` with `git stage`
As discussed in the team meeting, there is now a `git stage` command, which does exactly the same thing as `git add`, but has a slightly less confusing name (#61).

Let's just replace uses of `git add` with `git stage`. There a couple of caveats:

1. We should also amend the intermediate course, for consistency
2. Given that use of `git add` is still widespread, we should communicate to students that `git add` exists and is equivalent (both in the materials and verbally)

### PR DESCRIPTION
**Pull request by alexdewar**
I've amended the various places we use `git add` to use `git stage` instead. I've also added an information box to tell students that `git add` does the same thing.

Once this is merged I'll do a similar thing for the intermediate course.

Closes #62.

### CODE REVIEW

File: fig/git_areas.png
Comment (dc2917) [inline fig/git_areas.png:1]: Nice

**Review by dc2917 — COMMENTED** [review]
Looks good so far, nice catch with the figure too! There are a couple more uses of `git add` - one in the "Pushing an updated README" section of the remote repositories chapter and one in the summary section of the GitKraken chapter

**Review by AdrianDAlessandro — COMMENTED** [review]
This seems good to me, but Dan has pointed out some parts have been missed. So I won't approve yet

**Review by dc2917 — APPROVED** [review]

**Comment by dc2917** [comment]
> @dc2917 I can't see any other occurrences, except for these ones which all seem legit:
> 
> ```
> ❯ git grep -n 'git add'
> _episodes/l1-02-committing-history.md:209:  (use "git add <file>..." to include in what will be committed)
> _episodes/l1-02-committing-history.md:214:nothing added to commit but untracked files present (use "git add" to track)
> _episodes/l1-02-committing-history.md:251:> ## `git stage` vs `git add`
> _episodes/l1-02-committing-history.md:253:> Note that you will sometimes see the `git stage` command written as `git add` (e.g. in
> handouts/handout-lesson1.tex:59:   \item \textbf{git add} -- See ``git stage''
> ```

Yep, sorry, not sure how I missed that you'd changed the others 🙃

**Pull request merged**